pkgs/by-name/ai/airwindows/package.nix

```diff
@@ -0,0 +1,86 @@
+{
+  stdenv,
+  fetchFromGitHub,
+  fetchzip,
+  lib,
+  cmake,
+  nix-update-script,
+}:
+let
+  # adapted from oxefmsynth
+  vst-sdk = stdenv.mkDerivation {
+    dontConfigure = true;
+    dontPatch = true;
+    dontBuild = true;
+    dontStrip = true;
+    dontPatchELF = true;
+
+    name = "vstsdk3610_11_06_2018_build_37";
+    src = fetchzip {
+      url = "https://web.archive.org/web/20181016150224if_/https://download.steinberg.net/sdk_downloads/vstsdk3610_11_06_2018_build_37.zip";
+      sha256 = "0da16iwac590wphz2sm5afrfj42jrsnkr1bxcy93lj7a369ildkj";
+    };
+
+    installPhase = ''
+      cp -r VST2_SDK $out
+    '';
```

Comment:
Shouldn't this be packaged separately, rather than copy-pasted all over the place?

I'm finding _at least_ 3 other derivations which use it too.

Reply:
I don't think we're allowed to distrubute the VST2SDK, that's why I have it here.

Reply:
I'll consider making a VST2SDK package, wish there was a simple way of adding it to `buildInputs` but unfortunately anything using VST2 has it as a submodule :p